rust-lang/crates.io
# Move flash errors to service

## Tasks

~- [ ] Write automated tests to check flash errors behaviours and flows~ (moved to https://github.com/rust-lang/crates.io/issues/827)

## QA

- Make sure `set('flashError', …)` was replaced with `show(…)`
- Make sure `set('nextFlashError', …)` was replaced with `queue(…)`
- Manually test all of the flash error situations
